client/my-sites/customer-home/main.jsx

```diff
@@ -91,7 +91,7 @@ const Home = ( {
 						<Primary checklistMode={ checklistMode } />
 					</div>
 					<div className="customer-home__layout-col customer-home__layout-col-right">
-						<Secondary />
+						<Secondary cards={ layout.secondary } />
```

Comment:
I think we forgot to define `layout` here. Maybe land #40537 first and then rebase?

<img width="529" alt="Screen Shot 2020-03-31 at 3 21 56 PM" src="https://user-images.githubusercontent.com/1270189/78080578-63b53d00-7363-11ea-8dfc-44b201e8345a.png">